README.rst

```diff
@@ -33,34 +33,40 @@ Python est bienvenue et appréciée.
 En soumettant votre travail à la PSF pour inclusion dans la documentation,
 vous signifiez votre acceptation de cet accord.
 
+
 Contribuer à la traduction
 --------------------------
 
 Comment contribuer
 ~~~~~~~~~~~~~~~~~~
 
-Vous pouvez contribuer en utilisant :
+Vous pouvez contribuer en utilisant :
 
 - En envoyant un patch à la liste `traductions <https://lists.afpy.org/mailman/listinfo/traductions>`_.
 - Des *pull requests* Github (solution recommandée).
 
-Un guide pour contribuer via Github `se trouve ici 
-<https://github.com/python/python-docs-fr/CONTRIBUTION.rst>`_  :
+Un guide pour contribuer via Github `se trouve ici
+<https://github.com/python/python-docs-fr/CONTRIBUTION.rst>`_.
```

Comment:
Le lien me renvoie vers https://github.com/python/python-docs-fr/CONTRIBUTION.rst qui me dit « Not found »

Je suis moyennement content de ma suggestion, qui nécessitera une reprise à chaque fois qu'on changera de branche…

```suggestion
<https://github.com/python/python-docs-fr/blob/3.8/CONTRIBUTION.rst>`_.
```

Reply:
C'est fixé :)